R/choose_sites.R

```diff
@@ -0,0 +1,24 @@
+#' Identify sites with sufficient data to run a metabolism model
+#' 
+#' We will need to be more flexible later about which variables are available,
+#' since the file names will be variants on the core variable names (e.g.,
+#' wtr.local, wtr.model, etc.)
+#' 
+#' @param required_vars character vector of variable names that are all required
+#'   for a site to qualify
+#' @export
+#' @examples
+#' choose_sites()
+choose_sites <- function(required_vars=c("disch","doobs","wtr")) {
```

Comment:
yes, it's similar. i'm wondering, though, whether we'll continue to want to use this type of logic in mda.streams, since in the near future we won't be selecting sites based on what's available locally but more on what's available by local and/or proxy and/or modeled inputs.